build/zig/patches-0.11/hard-code-dynamic-linker.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/lib/std/target.zig b/lib/std/target.zig
+index c3d12b275..34e04f9aa 100644
+--- a/lib/std/target.zig
++++ b/lib/std/target.zig
+@@ -523,7 +523,6 @@ pub const Target = struct {
+                 .cloudabi,
+                 .dragonfly,
+                 .lv2,
+-                .solaris,
+                 .zos,
+                 .minix,
+                 .rtems,
+@@ -565,6 +564,7 @@ pub const Target = struct {
+                 .watchos,
+                 .driverkit,
+                 .shadermodel,
++                .solaris,
+                 => return .none,
+             }
+         }
+@@ -1572,7 +1572,7 @@ pub const Target = struct {
+             .netbsd => return copy(&result, "/libexec/ld.elf_so"),
+             .openbsd => return copy(&result, "/usr/libexec/ld.so"),
+             .dragonfly => return copy(&result, "/libexec/ld-elf.so.2"),
+-            .solaris => return copy(&result, "/lib/64/ld.so.1"),
++            .solaris => return copy(&result, "/usr/lib/amd64/ld.so.1"),
```

Comment:
What's the reason for this change?

Reply:
This is the path other executables use for the dynamic linker. So it seemed to me it's the preferred path to use to allow the system flexibility to link it to the correct interpreter.